tests/functional/test_model_completeness.py

```diff
@@ -10,16 +10,20 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+import json
+import os
 import pytest
 
 from botocore.exceptions import DataNotFoundError
 from botocore.loaders import Loader
-from botocore.session import Session
+
+loader = Loader()
+available_services = loader.list_available_services(type_name='service-2')
```

Comment:
Small nit; if variables are global constants you usually want to do the names in all caps (e.g. `LOADER`, `AVAILABLE_SERVICES`). Otherwise, you might look at using the [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) to adjust scoping if needed.

That would allow you to define something like:

```python
@pytest.fixture(scope="module")
def loader():
    return Loader()

@pytest.fixture(scope="module")
def available_services(loader):
    return loader.list_available_services(type_name='service-2')

[...]

 def test_all_endpoint_rule_sets_exist(loader, service_name, version):
     [...]
```